app/controllers/organization.js

```diff
@@ -217,7 +214,7 @@ module.exports = {
       console.log('isAdmin ', isAdmin)
       const allowedUpdates = ['settings', 'permissions', 'authentication']
       // if admin then check if valid update
-      if (isAdmin !== -1) {
+      if (isAdmin !== -1 || req.user.isAdmin === true) {
```

Comment:
do we need this here `isAdmin !== -1`

Reply:
I was checking into the organisation collection if that user is an admin or not. But without that too it will work fine using req.user.isAdmin === true.
So should I remove that.

Reply:
Done!